### PR TITLE
[Workers] Add clear route instructions

### PR DESCRIPTION
**Pull request by deadlypants1973**
PCX-6473

### CODE REVIEW

File: content/workers/platform/triggers/routes.md [hunk 1]
```diff
@@ -171,7 +175,7 @@ There is a well-known bug associated with path matching concerning wildcards (`*
 
 #### Domains and subdomains must have a DNS Record
 
-All subdomains must have a [DNS record](/dns/manage-dns-records/how-to/create-dns-records/) to be proxied on Cloudflare and used to invoke a Worker. For example, if you want to put a Worker on `myname.example.com`, and you have added `example.com` to Cloudflare but have not added any DNS records for `myname.example.com`, any request to `myname.example.com` will result in the error `ERR_NAME_NOT_RESOLVED`.
+All domains and subdomains must have a [DNS record](https://support.cloudflare.com/hc/en-us/articles/360019093151#h_60566325041543261564371) to be proxied on Cloudflare and used to invoke a Worker. For example, if you want to put a Worker on `myname.example.com`, and you have added `example.com` to Cloudflare but have not added any DNS records for `myname.example.com`, any request to `myname.example.com` will result in the error `ERR_NAME_NOT_RESOLVED`.
```
Comment (marciocloudflare) [inline content/workers/platform/triggers/routes.md]: Why did you add that `support.cloudflare.com` address? We aren't using this anymore

**Review by marciocloudflare — APPROVED** [review]

File: content/workers/platform/triggers/routes.md (hunk 1 above)
Comment (deadlypants1973) [inline content/workers/platform/triggers/routes.md]:
```suggestion
All domains and subdomains must have a [DNS record](/dns/manage-dns-records/how-to/create-dns-records/) to be proxied on Cloudflare and used to invoke a Worker. For example, if you want to put a Worker on `myname.example.com`, and you have added `example.com` to Cloudflare but have not added any DNS records for `myname.example.com`, any request to `myname.example.com` will result in the error `ERR_NAME_NOT_RESOLVED`.
```
